tests/test_ote_api.py

```diff
@@ -439,7 +439,7 @@ def train_and_eval(self, template_dir):
             export_performance = ov_task.evaluate(resultset)
             print(export_performance)
             performance_delta = export_performance.score.value - validation_performance.score.value
-            perf_delta_tolerance = 0.0005
+            perf_delta_tolerance = 0.01
```

Comment:
what if we make `perf_delta_tolerance` as the `export_perf_delta_tolerance`? It will allows us to track perf difference for atss and ssd, for their cases `perf_delta_tolerance` is quite good and should not be increased, I think

Reply:
wat?

Reply:
:)

Reply:
It makes sense. I'd say, much more than before.